specification/purview/data-plane/Azure.Analytics.Purview.MetadataPolicies/stable/2021-07-01/purviewMetadataPolicy.json

```
@@ -0,0 +1,535 @@
+        },
+        "attributeValueIncludes": {
+          "type": "string"
+        },
+        "attributeValueIncludedIn": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "attributeValueExcludes": {
+          "type": "string"
+        },
+        "attributeValueExcludedIn": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "DNFCondition": {
+      "type": "array",
+      "items": {
+        "type": "array",
+        "items": {
+          "$ref": "#/definitions/AttributeMatcher"
+        }
+      }
+    },
+    "CNFCondition": {
+      "type": "array",
+      "items": {
+        "type": "array",
+        "items": {
+          "$ref": "#/definitions/AttributeMatcher"
+        }
+      }
+    },
+    "DecisionRule": {
+      "type": "object",
+      "properties": {
+        "kind": {
+          "enum": [
+            "decisionrule"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "Kind",
+            "modelAsString": true
+          }
+        },
+        "effect": {
+          "enum": [
+            "Deny",
+            "Permit"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "Decision",
+            "modelAsString": true
+          }
+        },
+        "dnfCondition": {
+          "$ref": "#/definitions/DNFCondition"
+        }
+      }
+    },
+    "AttributeRule": {
+      "type": "object",
+      "properties": {
+        "kind": {
+          "enum": [
+            "attributerule"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "Kind",
+            "modelAsString": true
+          }
+        },
+        "id": {
+          "type": "string"
+        },
+        "name": {
+          "type": "string"
+        },
+        "dnfCondition": {
+          "$ref": "#/definitions/DNFCondition"
+        }
+      }
+    },
+    "CollectionReference": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "readOnly": true
+        },
+        "referenceName": {
+          "type": "string"
+        }
+      }
+    },
+    "MetadataPolicyProperties": {
+      "type": "object",
+      "properties": {
+        "description": {
+          "type": "string"
+        },
+        "decisionRules": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DecisionRule"
+          }
+        },
+        "attributeRules": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AttributeRule"
+          }
+        },
+        "collection": {
+          "$ref": "#/definitions/CollectionReference"
+        },
+        "parentCollectionName": {
+          "type": "string"
+        }
+      }
+    },
+    "MetadataPolicyViewModel": {
```

Simpler names are better.
```suggestion
    "MetadataPolicy": {
```